docs/install/README.md

```diff
@@ -220,18 +224,40 @@ $ pm2 start lib/index.js --name rsshub
 
 ### 添加配置
 
+::: tip 提示
+
+在 arm /arm64 上，此部署方式不包含 puppeteer 依赖。要启用 puppeteer，你需要先从发行版安装 Chromium，然后设置 `CHROMIUM_EXECUTABLE_PATH` 为其可执行路径。
```

Comment:
```suggestion
在 arm / arm64 上，此部署方式不包含 puppeteer 依赖。要启用 puppeteer，你需要先从发行版安装 Chromium，然后设置 `CHROMIUM_EXECUTABLE_PATH` 为其可执行路径。
```

Reply:
It was `docs/.format/format.js` that deleted the space.

Reply:
It is because `arm/arm64` is the right style.
FYI: https://zh-style-guide.readthedocs.io/zh_CN/latest/%E6%A0%87%E7%82%B9%E7%AC%A6%E5%8F%B7/%E5%B8%B8%E7%94%A8%E4%B8%AD%E6%96%87%E6%A0%87%E7%82%B9%E7%AC%A6%E5%8F%B7.html#id14

Reply:
I don't think `docs/.format/format.js` thinks so.
![](https://user-images.githubusercontent.com/15956627/166144031-d070fff5-e715-4ba3-883c-4d349915ba82.png)